### DISCUSSION TITLE
How to achieve pwa in a project using AppDirector

### DISCUSSION THREAD

**Discussion by lv-kit** [vercel/next.js#52383] [Help]
### Summary

I would like to realize pwa in a project using AppDirectory. However, the only information available is how to realize pwa with pagesDirectory.

If anyone knows how to realize pwa in a project using AppDirectory, I would appreciate your help!

**Comment by e-roy** [comment]
Looks like someone forked `next-pwa` and is regularly maintaining a newer version.

[https://www.npmjs.com/package/@ducanh2912/next-pwa](https://www.npmjs.com/package/@ducanh2912/next-pwa)

[https://github.com/DuCanhGH/next-pwa](https://github.com/DuCanhGH/next-pwa)


I'm having decent luck so far in development with the app router.

Hope this helps anyone else searching

**Reply by zacBkh** [reply]
Hi e-roy!
How is the development going so far?

I think the most updated way of doing would be using workbox right?
